src/snowflake/snowpark/_internal/data_source_utils.py

```diff
@@ -331,10 +329,6 @@ def generate_select_query(
         return f"""select {" , ".join(cols)} from {table}"""
     elif dbms == DBMS_TYPE.SQL_SERVER_DB or dbms == DBMS_TYPE.SQLITE_DB:
         return f"select * from {table}"
-    else:
-        raise NotImplementedError(
-            f"currently supported drivers are pyodbc and oracledb, got: {driver_info}"
-        )
```

Comment:
let's just do the following
if oracledb
 ...
else
   return f"select * from {table}"